CHANGELOG.md

```diff
@@ -47,6 +47,11 @@ The project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html)
 - Added hashed version of `Entry` called `ProofEntry`, which participates
   in the state aggregation. (#1553)
 
+#### exonum-cli
```

Comment:
Since some public interfaces were affected by this feature (namely, `Node::with_blockchain`), I think it should be mentioned in the `Breaking changes` section.